client-data.md

```diff
@@ -27,17 +23,42 @@ The default transfer encoding is Unixfsv1 with the following parameters:
 
 For details on how UnixfsV1 works, see its spec [here](https://github.com/ipfs/specs/tree/master/unixfs).
 
-## Storage Encoding
+## Storage Format
+
+The Storage Format it's the required format for generating Filecoin proofs and hashing sectors data. 
```

Comment:
This is not a sentence. Maybe rephrase it to 
`The Storage Format is the required format for generating Filecoin proofs and hashing sectors data. `

And again I would not put it as a proper name, not sure it adds real value to the table (it's not a product nor a feature etc)